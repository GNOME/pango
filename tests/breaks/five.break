<span segment='word'>abâ€¨cd</span>
