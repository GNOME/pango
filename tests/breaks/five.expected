Text:         ⁦a⁩ ⁦b⁩ [0x2028]  ⁦c⁩ ⁦d⁩  [0x0a]  
Breaks:     c  c c        Lc c c       Lc
Whitespace:      w             w       w 
Sentences:  bs   e        bs   e       b 
Words:      bs                 be      b 
Graphemes:  b  b b        b  b b       b 
Hyphens:       i             i           
